route_following_plugin/include/route_following_plugin.h

```diff
@@ -152,7 +145,7 @@ namespace route_following_plugin
         //Tactical plugin being used for planning lane change
```

Comment:
The config_limit variable above is really not needed at all. The usage of the config_limit and hardcoded max speed limit is handled inside carma_wm not need to cache it separately.